ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/private/cucushift-installer-rehearse-ibmcloud-ipi-private-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-private
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-ibmcloud-ipi-private-provision
+    post:
+    - chain: cucushift-installer-rehearse-ibmcloud-ipi-private-deprovision
+    env:
+      BASE_DOMAIN: "private-ibmcloud.qe.devcluster.openshift.com"
```

Comment:
The `BASE_DOMAIN` is already set in `cucushift-installer-rehearse-ibmcloud-ipi-private-provision`, so we do not need this env var, right?